src/connectors/radiofrance.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+Connector.playerSelector = '#player';
+
+Connector.pauseButtonSelector = '#player button.stopped';
+
+Connector.getTrackInfo = () => {
+	const artistText = Util.getTextFromSelectors('#player .line3');
+	const trackText = Util.getTextFromSelectors('#player .line2');
+
+	if (artistText && !artistText.includes(' - ') && trackText && !trackText.includes('Le direct')) {
+		return {
+			artist: artistText,
+			track: trackText,
+		};
+
+	} else if (artistText && artistText.includes(' - ')) {
+		return Util.splitArtistTrack(artistText);
+	}
+
+	return null;
+};
+
+Connector.getTrackArt = () => {
+	const trackArtUrl = Util.extractImageUrlFromSelectors('#player .cover img');
+
+	if (trackArtUrl && !trackArtUrl.includes('.svg')) {
+		return trackArtUrl.replace(/(?<=\/)\d+x\d+(?=_)/g, '400x400'); // larger image path
```

Comment:
Should we have a standard unit we aim to support? e.g. I saw in your prior one we resized to 300x300 which I copied in #3382

Reply:
For each of one I've submitted so far, I used values that were found in larger versions of the artwork located outside of the player elsewhere on the page/site, just to keep consistent with what they were already using.
I believe this one and Mixcloud were flexible with the value, so I considered using 272x272 (double the 136x136 size in the extension popup for HD screens). But some, like hoopla, have to be specific values or the image URL is no longer valid.
In answer to your question, I'm not sure if we need to set a standard or not—as long as the values are larger than 272x272 (maybe round up to 300) to keep the image crisp in the popup?